website/docs/cdktf/python/d/ec2_client_vpn_endpoint.html.markdown

```diff
@@ -0,0 +1,96 @@
+---
```

Comment:
Not a blocker, but previously we had discussed adding a comment header to each generated file stating that the file was generated and not to be edited. It would hopefully minimize contributors submitting PR's with direct edits.

Reply:
Yeah good point, I can include this in the next version 👍